data/watermarks/simple-text.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     x="0"
+     y="0">
+
+  <text x="0" y="5px" text-anchor="start"
+        fill="#$(WATERMARK_COLOR)"
+        font-family="$(WATERMARK_FONT_FAMILY)"
+        font-style="$(WATERMARK_FONT_STYLE)"
+        font-weight="$(WATERMARK_FONT_WEIGHT)"
+        font-size="10px">$(WATERMARK_TEXT)</text>
```

Comment:
No, don't want nor need font-size here. This is important as we need the half value of font-size in the y parameter. And sine anyway the svg is scaled to the whole image there is no need to make the code more complex here.